package.json

```diff
@@ -37,6 +37,7 @@
     "fortmatic": "^2.4.0",
     "preact": "^10.7.3",
     "qr-code-styling": "^1.6.0-rc.1",
+    "react-resize-detector": "^7.1.1",
```

Comment:
how much does it weigh in brotli format?

Reply:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/7966131/173096371-47e3b98a-3c30-4514-a2e6-58e842e7b544.png">